.github/workflows/test-code.yml

```
@@ -45,6 +45,13 @@ jobs:
           toolchain: nightly-2024-05-30
           command: test
 
+      - name: Test For runtime benchmarks
+        uses: actions-rs/cargo@v1
+        with:
+          toolchain: nightly-2024-05-30
+          command: test
+          args: --all --features runtime-benchmarks try-runtime
```

Don't we need to wrap it in quotation marks?
```suggestion
          args: --all --features "runtime-benchmarks try-runtime"
```

Also, can't we remove the other test in ll. 42-46 then?